jobs/base_model_validator.py

```diff
@@ -66,6 +67,30 @@ def process(self, input_model, regex_string):
             yield errors.ModelInvalidIdError(model)
 
 
+class ValidatePostCommitIsPrivate(beam.DoFn):
+    """DoFn to check if post_commmit_status is private when
+    post_commit_is_private is true and vice-versa.
+    """
+
+    def process(self, input_model):
+        """Function validates that post_commit_is_private is true iff
+        post_commit_status is private
+
+        Args:
+            input_model: base_models.BaseCommitLogEntryModel.
+                Entity to validate.
+
+        Yields:
+            ModelInvalidCommitStatus. Error for commit_type validation.
+        """
+        model = jobs_utils.clone_model(input_model)
+        expected_post_commit_is_private = (
+            model.post_commit_status ==
+            feconf.POST_COMMIT_STATUS_PRIVATE)
```

Comment:
```suggestion
            model.post_commit_status == feconf.POST_COMMIT_STATUS_PRIVATE)
```

Reply:
@vojtechjelinek I had to wrap this inside 80 chars.

Is there any other way to do this?

Reply:
I'm sorry I was a little confused earlier.

Reply:
Done